protocols/direct-funding.go

```diff
@@ -79,6 +79,8 @@ func NewDirectFundingObjectiveState(initialState state.State, myAddress types.Ad
 	}
 
 	init.FullyFundedThreshold = types.Funds{}
+	init.MyDepositSafetyThreshold = types.Funds{}
```

Comment:
I wonder if it is better/worse/the same as to write `make(types.Funds)` ? The `make` syntax allows us to initialize the size of the map, if we want to (it defaults to a "small" size).

Reply:
I wonder this as well.